app/stac_api/validators_view.py

```diff
@@ -38,7 +40,9 @@ def validate_item(kwargs):
         Http404: when the item doesn't exists
     '''
     if not Item.objects.filter(
-        name=kwargs['item_name'], collection__name=kwargs['collection_name']
+        Q(properties_expires=None) | Q(properties_expires__gte=timezone.now()),
+        name=kwargs['item_name'],
+        collection__name=kwargs['collection_name']
     ).exists():
         logger.error(
             "The item %s is not part of the collection %s",
```

Comment:
It might make sense to have a different error message in case that the item expired (though it is probably fine if the delete delay is only 24 hours)

Reply:
Could possibly do this. In my understanding from the user perspective the items are 'deleted' as soon as they are expired. In the admin page the expired items are still available (for 24 hours) and even editable to extend the expiration.